system-apps/password-reset-app/webClient/webpack.config.js

```diff
@@ -0,0 +1,66 @@
+
+
+/*
+  This program and the accompanying materials are
+  made available under the terms of the Eclipse Public License v2.0 which accompanies
+  this distribution, and is available at https://www.eclipse.org/legal/epl-v20.html
+  
+  SPDX-License-Identifier: EPL-2.0
+  
+  Copyright Contributors to the Zowe Project.
+*/
+
+var path = require('path');
+var webpackConfig = require('webpack-config');
+var CopyWebpackPlugin = require('copy-webpack-plugin');
+
+if (process.env.MVD_DESKTOP_DIR == null) {
+  throw new Error('You must specify MVD_DESKTOP_DIR in your environment');
+}
+
+var config = {
+  'entry': [
+    path.resolve(__dirname, './src/plugin.ts')
+  ],
+  'output': {
+    'path': path.resolve(__dirname, '../web'),
+    'filename': 'main.js'
+  },
+  'module': {
+    'rules': [
+      {'test': /\.scss$/,
+      'use': [
+  'exports-loader?module.exports.toString()',
+  {'loader': 'style-loader'},
+  {'loader': 'css-loader'},
+  {'loader': 'sass-loader'}
+              ],
+},
+    ]
+  },
+  'plugins': [
```

Comment:
you should add in the compress webpack plugin we use in the terminals & virtual-desktop plugins. it makes loading faster because the content is 1/3 the size when compressed.